pkg/ctl/create/nodegroup.go

```diff
@@ -30,6 +31,11 @@ func createNodeGroupCmd(cmd *cmdutils.Cmd) {
 			return api.ErrInvalidName(ng.Name)
 		}
 
+		if options.SubnetID != "" {
+			// TODO: validate the subnet being part of the VPC?
```

Comment:
mhmm we could do that, or let the natural CFN error bubble up.

Reply:
Oh, I actually do this already in another part of the validation. I guess I don't need this TODO any longer.

Reply:
https://github.com/weaveworks/eksctl/pull/4025/files#diff-84e0553c6ff9569d22f56a2a8038897508f76b733dd2304e73ee29f9c9ca1e69R608 herre